src/components/Dynamic.vue

```diff
@@ -44,6 +45,10 @@ export default {
       type: String,
       default: null,
     },
+    componentRef: {
+      type: String|Number,
+      default: null,
```

Comment:
won't the code break when the `componentRef` value is `null`? just to be sure the default value doesn't do damage

Reply:
nope no problem. Then there will be simply no ref attached.

Reply:
great :)